### PR TITLE
Sticky totals

### PR DESCRIPTION
**Pull request by teneightfive**
## User story [#99020392](https://www.pivotaltracker.com/story/show/99020392)

As a monies clerk I'd like the control total to remain visible at all times so it can be monitored

### CODE REVIEW

File: mtp_cashbook/assets-src/javascripts/modules/moj.sticky-header.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        if (!this.$stickyHeader.is(':visible')) {
+          this.$stickyHeader
+            .css({
+              top: -this.$originalHeader.height()
+            })
+            .show()
+            .stop()
+            .animate({
+              top: 0
+            }, 500);
```
Comment (SteveMarshall) [inline mtp_cashbook/assets-src/javascripts/modules/moj.sticky-header.js]: I assume the `500` here is half a second? Maybe worth pulling out into a variable?

File: mtp_cashbook/assets-src/javascripts/modules/moj.sticky-header.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        }
+      } else {
+        this.$stickyHeader
+          .stop()
+          .animate({
+            top: -this.$originalHeader.height()
+          }, 150, 'linear', function () {
```
Comment (SteveMarshall) [inline mtp_cashbook/assets-src/javascripts/modules/moj.sticky-header.js]: What's the `150` here?

File: mtp_cashbook/assets-src/javascripts/modules/moj.sticky-header.js (hunk 1 above)
Comment (teneightfive) [inline mtp_cashbook/assets-src/javascripts/modules/moj.sticky-header.js]: Yep, it's the animation duration. Not sure it's worth making that into another variable. The docs/examples for `.animate()` are clear enough to know what this is.

File: mtp_cashbook/assets-src/javascripts/modules/moj.sticky-header.js (hunk 2 above)
Comment (teneightfive) [inline mtp_cashbook/assets-src/javascripts/modules/moj.sticky-header.js]: Same as above, it's the duration for the animation.

Comment (SteveMarshall) [inline mtp_cashbook/assets-src/javascripts/modules/moj.sticky-header.js]: :+1:

**Comment by SteveMarshall** [comment]
Looks good to me, @teneightfive. @tonyduarte, you happy with it?

**Pull request merged**